boardswarm/Cargo.toml

```diff
@@ -45,3 +45,6 @@ reqwest = { version = "0.12.15", default-features = false, features = ["rustls-t
 strum = { version = "0.27.1", features = ["derive"] }
 strum_macros = "0.27.1"
 regex = "1.11.1"
+systemd-directories = "0.1.1"
+proc-mounts = "0.3.0"
+dirs = "6.0.0"
```

Comment:
2 crates just to get env variables?

Reply:
Yeah, another sign this might be completely wrong and should be a server configuration instead.

Reply:
not necessarilly ; but might be simpler to just implement it directly. though maybe erroring on simplicity and just require a absolute path in the provider config may be best. THe user will need to configure the mounting to happen there anyway

Reply:
Added as a provider parameter `mountpoint: String`
```
struct EswinEic7700StorageParameters {
    #[serde(rename = "match")]
    match_: HashMap<String, String>,
    mountpoint: String,
}
```

Reply:
Example configuration:
```
  - name: eswin-eic7700-storage
    provider: eswin-eic7700-storage
    parameters:                                                                 
      match:                                                                    
        udev.ID_VENDOR: "ESWIN"                                                 
        udev.ID_MODEL: "WIN-2030"                                               
      mountpoint: "/var/run/boardswarm/eswin-eic7700-storage"
```